code/pld/r2/r2020_cpu_glue/ic8_cpu_glue.pld

```diff
@@ -0,0 +1,33 @@
+GAL16V8    ; For ATF16V8BQL
+Glue       ; CPU glue logic for Classic 68020 edition
+
+Clock DS    SIZ0    SIZ1    A0    AS   NC    NC    NC   GND
+/OE   NC    LDS     UDS     NC    Q3   Q2    Q1    Q0   VCC
+
+Q0.R = /Q0
+
+
+Q1.R = /Q1 *  Q0
+     +  Q1 * /Q0
+
+
+Q2.R =  Q2 * /Q1
+     +  Q2 * /Q0
+     + /Q2 *  Q1 *  Q0
+
+
+Q3.R =  Q3 * /Q2
+     +  Q3 * /Q1
+     +  Q3 * /Q0
+     + /Q3 *  Q2 *  Q1 *  Q0
```

Comment:
I haven't tested this on hardware, but I figured out these equation in a spreadsheet, and they at least build for this PLD. `QX` is a no-connect pin that pulses once `Q2`-`Q0` get to either 4 or 2 (2 less than the cycle counts for each "half"-cycle), depending on whether `Q3` is low or high. On the next clock, that triggers the toggling of `Q3`, and resets the counter in `Q0`-`Q2`.

```suggestion
Clock DS    SIZ0    SIZ1    A0    AS   NC    NC    NC   GND
/OE   NC    LDS     UDS     QX    Q3   Q2    Q1    Q0   VCC

Q0.R = /QX * /Q0


Q1.R = /QX * /Q1 *  Q0
     + /QX *  Q1 * /Q0


Q2.R = /QX *  Q2 * /Q1
     + /QX *  Q2 * /Q0
     + /QX * /Q2 *  Q1 *  Q0


Q3.R = /QX *  Q3
     +  QX * /Q3


QX.R = /Q3 *  Q2 * /Q1 * /Q0
     +  Q3 * /Q2 *  Q1 * /Q0
```

Reply:
This looks good to me @0xTJ , and works great, thanks! As you say, probably minor (especially with full 6800 bus not really being supported on 68020) but good to stay in-spec as far as possible :) 

<img width="1545" alt="Screenshot 2022-04-20 at 06 40 22" src="https://user-images.githubusercontent.com/2096421/164158304-07006fff-bd25-4633-b2db-2240048b9794.png">